app/javascript/Components/__tests__/marks_panel.test.jsx

```diff
@@ -153,6 +152,10 @@ describe("CheckboxCriterionInput", () => {
 
     expect(screen.queryByText(`(${I18n.t("results.remark.old_mark")}: 1)`)).toBeTruthy();
   });
+
+  it("renders CheckboxCriterionInput", () => {
+    render(<CheckboxCriterionInput {...basicProps} />);
```

Comment:
You should improve these tests by looking for particular elements on the screen. It's not enough to just call `render` with the components.